src/pages/ProjectsPage.js

```diff
@@ -75,8 +74,11 @@ const SelectedCard = ({ project, setSelected }) => {
         <h5 className="pt-3"> About the Project </h5>
         <p>{projectAbout}</p>
 
-        <h5 className="pt-3"> Project Timeline </h5>
-        <p className="card-text">{projectTime}</p>
+        {projectDeadline && (
+          <>
+            <h5 className="pt-3"> Project Timeline </h5>
+            <p className="card-text">{projectDeadline}</p>
+          </>)}
```

Comment:
i'm wondering if not rendering the component instead of adding a default text might be confusing to people, but this is super minor and can be evaluated later if need be :)